react-charts/sparkline-chart/step4.md

```diff
@@ -1,36 +1,23 @@
-Now we'll add tooltips to the sparkline chart...
+Add a label to the sparkline chart.
 
-## Task
+1) <strong>Make sure `App.js` is still open.</strong>
 
-1) Make sure the App.js file is still open
-
-2) Locate the code that looks like the following:
+2) <strong>Locate the code that looks like this:</strong>
 
 <pre class="file">
-&lt;ChartGroup
-  maxDomain={{y: 9}}
-  padding={0}
-  height={100}
-  width={400}
-&gt;
+&lt;div className=&quot;chart-container&quot;&gt;
+  ...
+&lt;/div&gt;
 </pre>
 
-3) Add the following property to that section:
+3) Add the following code inside the `<div>`

```

Comment:
```suggestion
3) Add the `ChartContainer` component outside of the wrapper identified in step 2.
```

(@dlabrecq can you confirm that this is right)

Reply:
And then add a step for adding the Chart Label inside of the Chart Container. also explain why you need a chart container component

Reply:
i think its safe to commit the suggestion above ^ @wojta

Reply:
>  Add the `ChartContainer` component outside of the wrapper identified in step 2.
outside?

Reply:
Yep can confirm it sits below the identified section:
This is from the original Sparkline

<img width="408" alt="Screen Shot 2019-12-10 at 11 43 13 AM" src="https://user-images.githubusercontent.com/20118816/70549481-49501d00-1b42-11ea-85b1-619288417de9.png">